tableauserverclient/server/server.py

```diff
@@ -76,7 +77,7 @@ def _get_legacy_version(self):
     def _determine_highest_version(self):
         try:
             old_version = self.version
-            self.version = "2.4"
+            self.version = "3.0"
```

Comment:
Don't change this one.  The endpoint we are using to check the highest version was introduced in 2.4, so we need to force it to use that version to cover old versions.